package.json

```diff
@@ -78,22 +79,19 @@
   },
   "devDependencies": {
     "@1stg/browserslist-config": "^1.2.3",
-    "@1stg/commitlint-config": "^3.1.4",
-    "@1stg/lint-staged": "^3.3.0",
-    "@1stg/remark-config": "^4.0.3",
-    "@1stg/simple-git-hooks": "^0.2.1",
     "@1stg/stylelint-config": "^5.0.0",
     "@babel/core": "^7.16.0",
     "@babel/eslint-parser": "^7.17.0",
     "@babel/plugin-proposal-function-bind": "^7.16.7",
     "@babel/types": "^7.16.0",
     "@changesets/changelog-github": "^0.4.6",
     "@changesets/cli": "^2.24.2",
+    "@changesets/get-release-plan": "^3.0.16",
```

Comment:
If we want to use some dependency in our code, `pnpm` requests us to add these deps explicitly. That's why I added a few deps.